build.gradle

```diff
@@ -6,6 +6,14 @@ plugins {
     id "com.github.johnrengelman.shadow" version "5.0.0"
     id 'maven-publish'
     id 'net.researchgate.release' version '2.6.0'
+    id 'jacoco'
+}
+
+repositories {
```

Comment:
We already have this in respositories.gradle file. We can import and reuse them.

Reply:
I will do it

Reply:
Commit id is e02d20457833732d101b1114a49aa306735515a7